R/derive_merged.R

```diff
@@ -336,11 +377,24 @@ derive_vars_merged_dt <- function(dataset,
   filter_add <- assert_filter_cond(enquo(filter_add), optional = TRUE)
   assert_data_frame(dataset_add, required_vars = quo_c(by_vars, dtc))
 
+  deprecate_warn(
+    "0.8.0",
+    "derive_vars_merged_dt()",
+    details = "Please use `derive_vars_dt()` and `derive_vars_merged()` instead."
+  )
+
   old_vars <- names(dataset_add)
+  if (is.null(date_imputation)) {
+    highest_imputation <- "n"
+    date_imputation <- "first"
+  } else {
```

Comment:
Change keyword to deprecated so it removes it from our general reference page and into the deprecated section

Reply:
same for other deprecated functions in this PR

Reply:
Updated